src/main/java/io/airlift/airline/GlobalSuggester.java

```diff
@@ -16,10 +17,11 @@ public class GlobalSuggester
     @Override
     public Iterable<String> suggest()
     {
-        return concat(
-                concat(
+        Iterator<String> result = Streams.concat(
+                Streams.concat(
                         metadata.getCommandGroups().stream().map(CommandGroupMetadata::getName),
                         metadata.getDefaultGroupCommands().stream().map(CommandMetadata::getName)),
-                metadata.getOptions().stream().flatMap(option -> option.getOptions().stream()))::iterator;
+                metadata.getOptions().stream().flatMap(option -> option.getOptions().stream())).iterator();
+        return (Iterable<String>) result;
```

Comment:
I don't think you can just cast the result from an iterator to an iterable. They are different types. Iterables can produce iterators, but Iterators provide actual items to iterate over